dashboard/app/models/sections/email.rb

```diff
@@ -0,0 +1,29 @@
+# == Schema Information
+#
+# Table name: sections
+#
+#  id                :integer          not null, primary key
+#  user_id           :integer          not null
+#  name              :string(255)
+#  created_at        :datetime
+#  updated_at        :datetime
+#  code              :string(255)
+#  script_id         :integer
+#  course_id         :integer
+#  grade             :string(255)
+#  login_type        :string(255)      default("email"), not null
+#  deleted_at        :datetime
+#  stage_extras      :boolean          default(FALSE), not null
+#  section_type      :string(255)
+#  first_activity_at :datetime
+#  pairing_allowed   :boolean          default(TRUE), not null
+#
+# Indexes
+#
+#  fk_rails_20b1e5de46        (course_id)
+#  index_sections_on_code     (code) UNIQUE
+#  index_sections_on_user_id  (user_id)
+#
+
+class Email < Section
```

Comment:
Good point. This has always bugged me about our STI levels, which by name seem to have no relation. The path does help, so all the files are under the `levels` (or `sections` in this case) directory.

Maybe we could suffix each type  with `Section` and add that to the string in `Section.find_sti_class`, so e.g. `email` would map to `EmailSection`